pkg/payload/precondition/clusterversion/recommendedupdate.go

```diff
@@ -88,29 +92,34 @@ func (ru *RecommendedUpdate) Run(ctx context.Context, releaseContext preconditio
 			Reason: "NoChannel",
 			Message: fmt.Sprintf("Configured channel is unset, so the recommended status of updating from %s to %s is unknown.",
 				clusterVersion.Status.Desired.Version, releaseContext.DesiredVersion),
-			Name: ru.Name(),
+			Name:               ru.Name(),
+			NonBlockingWarning: true,
 		}
 	}
 
 	reason := "UnknownUpdate"
 	msg := ""
-
+	warning := false
```

Comment:
no need for this variable, you can just hard-code `NonBlockingWarning: true` belown in the `return` line.

Reply:
Resolved in https://github.com/openshift/cluster-version-operator/commit/481bcde393f244f4e4245ebec6a43941c87626de